portal.properties

```diff
@@ -183,7 +182,10 @@ recache_study_after_update=false
 # excluding this value or setting it to an empty string will revert to the previous bookmarking method
 # WARNING: do not use session service with -Dauthenticate=false
 #  either use authentication or change to -Dauthenticate=noauthsessionservice
-session.service.url=
+session.service.url=http://cbio-session-service:8080/api/sessions/main_session/
```

Comment:
Does this need to be here, if it's also supplied as a JVM property? Do the two need to match, and if so, could you add a comment in both places so that we don't forget that if we do change the URL?

Reply:
We just pair-programmed to phrase a comment that would clarify where and when this is configured 👍